UKHO.D365CallbackDistributorStub.API/UKHO.D365CallbackDistributorStub.API/Services/CallbackService.cs

```diff
@@ -1,21 +1,33 @@
 ﻿using System.Diagnostics.CodeAnalysis;
+using System.Net;
 using UKHO.D365CallbackDistributorStub.API.Models.Request;
 
 namespace UKHO.D365CallbackDistributorStub.API.Services
 {
     [ExcludeFromCodeCoverage]
-    public class CallbackService
+    public class CallbackService 
     {
         private static readonly Queue<RecordCallbackRequest> s_recordCallbackRequestQueue = new();
-        public static bool SaveCallbackRequest(CallbackRequest callbackRequest, string subscriptionId)
+        private static readonly List<CommandCallbackRequest> s_commandCallbackRequestList = new();
+        private const HttpStatusCode _noContent = HttpStatusCode.NoContent;
```

Comment:
Private consts should be Pascal case please:
```suggestion
        private const HttpStatusCode NoContent = HttpStatusCode.NoContent;
```

Reply:
Changes done